lib/kino_explorer/data_transform_cell.ex

```diff
@@ -967,11 +973,6 @@ defmodule KinoExplorer.DataTransformCell do
 
   defp implements?(protocol, value), do: protocol.impl_for(value) != nil
 
-  defp is_data_frame?(ctx) do
-    df = ctx.assigns.root_fields["data_frame"]
-    Map.get(ctx.assigns.data_frame_variables, df)
```

Comment:
Sidenote: the reason for the linked issue is that here, if we have no variable information, we would return `nil`, which fails the guard `defp lazy(nodes, %{is_data_frame: false}), do: nodes`, and consequently produces a read-lazy code, which may not be applicable.